composer.json

```diff
@@ -1,35 +1,35 @@
-{
-    "name": "common-gateway/zgw-bundle",
-    "description": "Symfony bundle for zgw standard functionality",
-    "type" : "symfony-bundle",
-	"keywords" : [
-		"commongateway",
-		"common",
-		"gateway",
-		"conduction",
-		"symfony",
-		"common-gateway-plugin",
-        "zgw"
-	],
-	"homepage" : "https://commongateway.nl",
-    "license": "EUPL-1.2",
-    "minimum-stability" : "dev",
-    "require": {
-        "php": ">=7.4",
-		"commongateway/corebundle": ">=1.0.79",
-		"common-gateway/referentielijsten-bundle": ">=0.1.0"
-    },
-	"require-dev" : {
-		"symfony/dependency-injection" : "~3.4|~4.1|~5.0"
-	},
-	"autoload" : {
-		"psr-4" : {
-			"CommonGateway\\ZGWBundle\\" : "src/"
-		}
-	},
-    "scripts": {
-        "post-package-install": "bin/console commongateway:install common-gateway/zgw-bundle",
-        "post-package-update": "bin/console commongateway:update common-gateway/zgw-bundle",
-        "post-package-uninstall": "bin/console commongateway:uninstall common-gateway/zgw-bundle"
-    }
-}
+{
+    "name": "common-gateway/zgw-bundle",
+    "description": "Symfony bundle for zgw standard functionality",
+    "type" : "symfony-bundle",
+	"keywords" : [
+		"commongateway",
+		"common",
+		"gateway",
+		"conduction",
+		"symfony",
+		"common-gateway-plugin",
+        "zgw"
+	],
+	"homepage" : "https://commongateway.nl",
+    "license": "EUPL-1.2",
+    "minimum-stability" : "dev",
+    "require": {
+        "php": ">=7.4",
+		"commongateway/corebundle": "^1.1.110 | <2.0"
+		"common-gateway/referentielijsten-bundle": ">=0.1.0 | dev-feature/GW-1596/Symfony-updates"

```

Comment:
You still have a dev branche here, is this on purpose?

Reply:
No, should remove this